### PR TITLE
restructure english content and initialize japanese localization

### LINKED ISSUE #145: Plans for localization
Hi team

Some of the Japanese members are interested in localizing the TAG runtime materials (webpage and white papers).
Are there any plans to do a localization?

### PR DESCRIPTION
**Pull request by kaitoii11**
Closes #145

As I mentioned in #145, I have reorganized some content and added minimal Japanese localization for the website.
Let me know what the team thinks and I would love to have some feedback from the maintainers.

As for the Japanese localization, I will ask the Japanese members who have experience localizing CNCF glossary.

### CODE REVIEW

**Comment by kaitoii11** [comment]
PTAL @raravena80

**Review by raravena80 — APPROVED** [review]
Thank you. Looks like there are still quite a bit of pages that need to be translated but this is a great start! 😀

**Comment by kaitoii11** [comment]
PTAL @rajaskakodkar

File: website/content/ja/_index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: "CNCF TAG ランタイム"
```
Comment (Okabe-Junya) [inline website/content/ja/_index.md:2]:
```suggestion
title: "CNCF TAGランタイム"
```

https://github.com/cncf/tag-runtime/blob/f40f28fe0f2af1122f30a89884a4e79b7b2af03e/website/config.toml#L58 に準拠すると半角スペース抜きですかね。。？？

File: website/content/ja/search.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: 検索結果
```
Comment (Okabe-Junya) [inline website/content/ja/search.md:2]: glossaryでは "Search Result"のママ表記でしたね。。（決めの問題なのでどっちでも良いとは思いますが、この辺りはcncf配下では表記を統一したいなと思っています）

https://github.com/cncf/glossary/blob/c12d328a4bc7901ce334b41055ec38724e8ac101/content/ja/search.md?plain=1#L2

File: website/content/ja/wgs/cnaiwg/_index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: クラウドネイティブ AI ワーキンググループ
```
Comment (Okabe-Junya) [inline website/content/ja/wgs/cnaiwg/_index.md:2]:
```suggestion
title: クラウドネイティブAIワーキンググループ
```

File: website/content/ja/wgs/cnaiwg/mission_vision_strategy/index.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
+---
+title: ミッション、 ビジョン、および主要成果物
```
Comment (Okabe-Junya) [inline website/content/ja/wgs/cnaiwg/mission_vision_strategy/index.md:2]:
```suggestion
title: ミッション、ビジョン、および主要成果物
```

File: website/content/ja/wgs/cnaiwg/whitepapers/cloudnativeai.md [hunk 5]
```diff
@@ -0,0 +1,694 @@
+---
+title:  "CNCF Cloud Native AI White Paper"
```
Comment (Okabe-Junya) [inline website/content/ja/wgs/cnaiwg/whitepapers/cloudnativeai.md:2]: Just commentですが、半角2つ入っているの気になりますね。。

File: website/content/ja/wgs/cnaiwg/whitepapers/schedulingai/index.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+---
+title:  "Scheduling AI workloads on K8s TBD"
```
Comment (Okabe-Junya) [inline website/content/ja/wgs/cnaiwg/whitepapers/schedulingai/index.md:2]: この辺りも何故かスペース2こ入ってますよね。。

File: website/content/ja/wgs/search.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+---
+title: 検索結果
```
Comment (Okabe-Junya) [inline website/content/ja/wgs/search.md:2]: ditto

File: website/content/ja/wgs/wasm/_index.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+---
+title: WebAssembly (WASM) ワーキンググループ
```
Comment (Okabe-Junya) [inline website/content/ja/wgs/wasm/_index.md:2]:
```suggestion
title: WebAssembly(WASM)ワーキンググループ
```

File: website/content/ja/wgs/wasm/charter/index.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
+---
+title: WebAssembly (Wasm) ワーキンググループチャーター
```
Comment (Okabe-Junya) [inline website/content/ja/wgs/wasm/charter/index.md:2]:
```suggestion
title: WebAssembly(Wasm)ワーキンググループチャーター
```

File: website/content/ja/wgs/wasm/deliverables/_index.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
+---
+title: WASM ワーキンググループ成果物
```
Comment (Okabe-Junya) [inline website/content/ja/wgs/wasm/deliverables/_index.md:2]:
```suggestion
title: WASMワーキンググループ成果物
```

File: website/config.toml
Comment (Okabe-Junya) [inline website/config.toml:1]: （コメントですが）

TAG、WG、Charterなどの翻訳 / 表記方法だけ全体で統一したいですよね。。「`TAG Runtime` など、`TAG hoge` はhogeの部分だけ訳す」方針で良さそうですかね？

そしてその辺りのl10nルールをcncf配下で統一できると嬉しいんですが、明記する上で良さそうな場所ありますかね。。

**Review by Okabe-Junya — COMMENTED** [review]
let me review from the perspective of Japanese l10n as one of the Japanese l10n team members

Note that I did not review the directory structure or anything related to Hugo.

**Review by rajaskakodkar — APPROVED** [review]
This is a great start and thank you so much!

**Comment by Okabe-Junya** [comment]
Three people have approved this PR. I think this is sufficient LGTM to merge this PR!!

WDYT @raravena80??

**Comment by nikhita** [comment]
Love all the progress here 🙌

**Comment by raravena80** [comment]
@kaitoii11 can you sign the DCO? Thank you!

**Comment by kaitoii11** [comment]
Is there anything from my end that needs to be done to move this forward?

**Review by raravena80 — APPROVED** [review]
lgtm!

**Pull request merged**